src/renderer/util.ts

```diff
@@ -161,3 +161,8 @@ export const formatDate: (date: Date) => string = (date) => {
 
   return [dd, mm, yy].map(pad).join('/');
 };
+
+export const removeExtension: (fileName: string) => string = (fileName) => {
+  const split = fileName.split('.');
+  return split.slice(0, split.length - 1).join('.');
```

Comment:
Is there a necessity to do `.join('.')` instead of `.join('')`? 

They seem to do the same thing:

![image](https://user-images.githubusercontent.com/55727741/168572999-016fb34a-b9bc-4cbc-86e2-c3654729188e.png)

Reply:
yeah because if the file has multiple dots 

e.g.

someproject.2022.mlvet

would change to 

someproject2022

if we join('')

edge case but may as well account for it, dots in file names are allowed